run_all_experiments.py

```diff
@@ -50,6 +50,7 @@
 BENCHMARK_DIR: str = f'{BENCHMARK_ROOT}/comparison'
 RESULTS_DIR: str = run_one_experiment.RESULTS_DIR
 GENERATED_BENCHMARK: str = 'generated-benchmark-'
+PROJECT_SUMMARY_JSON = os.path.join(RESULTS_DIR, 'project_summary.json')
```

Comment:
We shouldn't be hardcoding `RESULTS_DIR` here. We should instead be using `args.work_dir` here.

Reply:
I think that is unrelated to this PR, I just follows the original approach. Maybe we should do that in a separate PR?? WDYT? @oliverchang

Reply:
If I'm reading this PR correctly, this misuse is newly introduced. 

`RESULTS_DIR`` here on line 51 is simply used as a default value for argparse here:  https://github.com/google/oss-fuzz-gen/blob/d238872f20946a55d0c0de40adbaea39c1918510/run_all_experiments.py#L224

We should not be using this directly.